erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -31,7 +31,8 @@ frappe.ui.form.on("BOM", {
 			return {
 				query: "erpnext.manufacturing.doctype.bom.bom.item_query",
 				filters: {
-					"is_stock_item": 1
+					"is_stock_item": 1,
+					"is_template_item": 1
```

Comment:
```suggestion
```

This seems confusing. Should we just implicitly `OR` this intead? Because template items are only an exception to it.